### PR TITLE
polyfills are correctly resolved now, but some crypto polyfill is not implemented yet

### PR DESCRIPTION
**Pull request by clangenb**
Blocker: https://github.com/unjs/unenv/issues/177

Currently getting the following error during encryption:

```console
 Error: Error during encryption. Original error: Error: [unenv] crypto.createHash is not implemented yet!
```

Workarounds:
* (again) try to make the vite-polyfils-plugin working (or maybe a combination of unenv/vite), created bug report: https://github.com/davidmyersdev/vite-plugin-node-polyfills/issues/92
* migrate crypto in encointer-js to another lib
* maybe the node-rsa v1.1.2 with its own polyfills can resolve the issue: https://github.com/rzcoder/node-rsa/issues/234
* ~~use `clientNodeCompat: true`:~~  won't work, it uses unenv under the hood, see #11

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,13 +21,12 @@
...
     "postcss": "^8.4.38",
     "tailwindcss": "^3.4.3",
-    "unenv": "^1.9.0",
     "vite": "^5.2.10",
     "vue": "^3.4.26",
     "vue-router": "^4.3.2"
   },
   "dependencies": {
-    "@encointer/worker-api": "^0.12.4",
+    "@encointer/worker-api": "^0.12.5-alpha.0",
```
Comment (clangenb) [inline package.json:29]: doesn't fix anything

**Comment by brenzi** [comment]
there are so many "unimplented" calls in unenv node crypto that I fear this rabbit hole is deep. simply implementing crateHash will likely just lead us to the next missing piece

**Comment by brenzi** [comment]
AFAIU, polkadot-api uses wasm for crypto stuff. I guess they didn't implement RSA, but they clearly implemented hashing. could we just use polkadot api to do the missing stuff?

**Comment by brenzi** [comment]
better docs on NaCl asymmetric encryption: 
https://pynacl.readthedocs.io/en/latest/public/

tracking issue: https://github.com/integritee-network/worker/issues/1597

**Comment by clangenb** [comment]
No longer needed, fixed crypto with #13

**Pull request closed without merging**